tagbot/action/repo.py

```diff
@@ -179,8 +179,9 @@ def _registry_pr(self, version: str) -> Optional[PullRequest]:
             return None
         name = self._project("name")
         uuid = self._project("uuid")
+        sha = self.commit_sha_of_version(version)
```

Comment:
Is this correct? I thought this was the hash of the URL, not the hash of the version number.

Reply:
(to clarify: just using the GitHub repo's URL would work for >95% of packages and take almost no effort, looking it up in the registry would always be correct but requires writing some more code)

Reply:
ok, thanks. If you have a clear idea here please feel free to push. Otherwise I can try to figure it out. (I'm not that comfortable in this code base)

Reply:
Ok, gave it a go

Reply:
Woo! Green